conans/client/command.py

```diff
@@ -174,41 +177,38 @@ def test(self, *args):
                                 build_modes=args.build, test_build_folder=args.test_build_folder)
 
     def create(self, *args):
-        """ Builds a binary package for recipe (conanfile.py) located in current dir.
-        Uses the specified configuration in a profile or in -s settings, -o options etc.
-        If a 'test_package' folder (the name can be configured with -tf) is found, the command will
-        run the consumer project to ensure that the package has been created correctly. Check the
-        'conan test' command to know more about the 'test_folder' project.
+        """Builds a binary package for a conanfile.py. Uses the specified configuration in a
```

Comment:
actually a package can be created from a ``custom_file.py`` conanfile. => ``from a conanfile``